base/applications/cmdutils/replace/util.c

```diff
@@ -0,0 +1,275 @@
+/*
+ * PROJECT:     ReactOS Command shell
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Implements 'replace' command
+ * COPYRIGHT:   Copyright Samuel Erdtman (samuel@erdtman.se)
+ * COPYRIGHT:   Copyright 2024 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
+
+#include "replace.h"
+
+BOOL bCtrlBreak = FALSE;  /* Ctrl-Break or Ctrl-C hit */
+
+INT
```

Comment:
```suggestion
/*
 * Helper function for getting the current path from drive
 * without changing the drive. Return code: 0 = ok, 1 = fail.
 * 'InPath' can have any size; if the two first letters are
 * not a drive with ':' it will get the current path on
 * the current drive exactly as GetCurrentDirectory() does.
 */
INT
```
keeping the documentation as it is in the cmd file too.